examples/device/board_test/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ set(TOP "../../..")
 get_filename_component(TOP "${TOP}" REALPATH)
 
 # Check for -DFAMILY=
-if(FAMILY STREQUAL "esp32s2")
+if(FAMILY MATCHES "^(esp32s[2-3])*")
```

Comment:
regex is not quite right, `*` allow some odds such as `esp32s2esp32s3` match as well. We could simply have it as `"^esp32s[2-3]"`

Reply:
> 
> 
> regex is not quite right, `*` allow some odds such as `esp32s2esp32s3` match as well. We could simply have it as `"^esp32s[2-3]"`

agree, in some very rare cases it can cause some issues.